src/app/_css/globals.css

```diff
@@ -26,6 +25,12 @@
   --dark-rock-800: #403F4C;
   --soft-white-100: #F6F6F6;
 
+  /* Links */
+  a {
```

Comment:
Btw, there are some default styling behaviours of browsers which are a pain. A good idea is to use a "CSS reset" (like this one https://www.joshwcomeau.com/css/custom-css-reset/), it helps to have more of a blank slate to work with.

Reply:
this is excelent! Thanls for sharing